.github/workflows/mutation_tests.yml

```diff
@@ -0,0 +1,24 @@
+name: Mutation tests
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
```

Comment:
By default is it 24 format time? It will be 02:00PM?

Reply:
yes, 24 format 
02:00 **AM**